Script/AtomicEditor/ui/frames/menus/MainFrameMenu.ts

```diff
@@ -40,7 +40,8 @@ class MainFrameMenu extends Atomic.ScriptObject {
         MenuItemSources.createMenuItemSource("menu tools", toolsItems);
         MenuItemSources.createMenuItemSource("menu developer", developerItems);
         MenuItemSources.createMenuItemSource("menu help", helpItems);
-
+        this.goScreenshot = 0;
+        this.subscribeToEvent(Atomic.UpdateEvent((ev) => this.handleScreenshot(ev)));
```

Comment:
@JoshEngebretson that's definitely doable, but it's only built in if you add the ```require("AtomicEventLoop")``` to expose it.  Is there a way to auto-load that from the native side so that it's always available and doesn't have to be required in?  If that's doable then adding the pieces to ```duktape.d.ts``` can be done and that functionality used w/o needing any additional setup.